### PR TITLE
Update User Guide & Readmes

### PR DESCRIPTION
**Pull request by Jake-Carter**
* Add Cordio BLE documentation to User Guide
* Update all READMEs with links to the online User Guide
* Update all READMEs with project-specific build notes
* General documentation clean up

Preview:  https://jake-carter.github.io/msdk/

### CODE REVIEW

File: Libraries/Cordio/docs/Applications/Bootloader.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Description
```
Comment (kevin-gillespie) [inline Libraries/Cordio/docs/Applications/Bootloader.md:1]: These README files can stay in place. This isn't specifically a Cordio application.

Comment (kevin-gillespie) [reply Libraries/Cordio/docs/Applications/Bootloader.md:1]: By these I mean the Bootloader files.

File: Libraries/Cordio/docs/CORDIO_USERGUIDE.md [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+
+Once a initiating devices sends the connection requset and the advertising devices accepts the connection request, the two devices enter the connected state. This is a point to point connection allowing devcies to directly exchange information. 
+
+In order to minimize interference, devices in the connected state will hop between channels 0-36 in a pseudo random order. The channel hopping information is communicated in the connection indication.
+
+Each connection event is separated by the connection interval. This interval is configurable from 7.5 ms to 4 s. The master will always transmit first and receive second. The Slave will always receive first and transmit second. Devices will typically always send and receive at least one packet in each interval, and they can optionally transmit and receive multiple packets in each interval. 
+
+![Connected](res/Connected.png)
+
+## Packetcraft Protocol Software
```
Comment (kevin-gillespie) [inline Libraries/Cordio/docs/CORDIO_USERGUIDE.md:141]: We should move the Packetcraft documentation to a separate file. Basically everything from here to Platforms. This doesn't need to be in our UG.

File: Libraries/Cordio/README.md [hunk 3]
```diff
@@ -1,84 +1,25 @@
-Packetcraft Protocol Software
-=============================
+# Cordio
+This library implements the Bluetooth Low Energy stack , forked from [Packetcraft](https://github.com/packetcraft-inc/stacks). It depends on additional libraries within the msdk repository. All of the applications utilizing this library can be found in the [Examples](../../Examples) directory.
 
-Packetcraft protocol software is a collection of embedded stacks implementing the Bluetooth Low Energy Link Layer, Host, Profile and Mesh specification (www.bluetooth.org).
+## Getting Started
```
Comment (kevin-gillespie) [inline Libraries/Cordio/README.md:4]: Can we include the contents of this README in the UG?

**Review by kevin-gillespie — CHANGES_REQUESTED** [review]
We still have "MAX32655" in the platform documentation pdf.

File: Libraries/Cordio/docs/CORDIO_USERGUIDE.md [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+    FW-3736 Light CTL Model
+    FW-3738 Mesh v1.0.1 compliant
+    FW-3739 TCRL.2019-2 qualification
+    FW-3750 Laird BL654 platform
+    FW-3767 SBC codec
+    FW-3803 GCC compiler support for gcc-arm-none-eabi-9-2019-q4-major
+    FW-3820 Protect against SweynTooth vulnerability
+
+
+Getting Started
```
Comment (Jake-Carter) [inline Libraries/Cordio/docs/CORDIO_USERGUIDE.md:170]: This section recommends manually installing GCC and has some broken links.  We should remove this entirely or rework it to point to the MSDK Getting Started sections and highlight some good BLE starter projects.

File: Libraries/Cordio/docs/CORDIO_USERGUIDE.md [hunk 5]
```diff
@@ -0,0 +1,276 @@
...
+
+This command will build a complete device image including the Tag sample application, Profiles, Host, Link Layer and platform drivers for the Nordic nRF52840 / PCA10056 development board. The resulting image is located at `ble-apps/build/tag/gcc/bin/tag.bin`.
+
+
+**3. Program**
+
+To install the firmware image on a Nordic PCA10056 developmnent board, plug a USB cable into your PCA10056. Drag-n-Drop the resulting image from the previous step onto the mass storage drive called "JLINK".
+
+
+Documentation
```
Comment (Jake-Carter) [inline Libraries/Cordio/docs/CORDIO_USERGUIDE.md:210]: Remove this section, this _is_ the documentation

**Review by AbbyWolf-ADI — APPROVED** [review]

**Pull request merged**